front/pages/[user]/a/[sId]/clone.jsx

```diff
@@ -119,7 +116,7 @@ export default function CloneView({ app, user, ga_tracking_id }) {
                       </label>
                       <div className="mt-1 flex rounded-md shadow-sm">
                         <span className="inline-flex items-center rounded-l-md border border-r-0 border-gray-300 bg-gray-50 pl-3 pr-1 text-sm text-gray-500">
-                          {session.user.username}
+                          {authUser.username}
```

Comment:
clone is the unique place where we use authUser. Which makes sense as we need a destination :)